reportengine.php

```diff
@@ -215,8 +215,8 @@
 			foreach ($options as $option) {
 				$opt                   = explode('=>', $option);
 				list($value, $display) = $opt;
-				if (preg_match('/^I18N::number\((.+)\)$/', $display, $match)) {
-					$display = I18N::number($match[1]);
+                		if (preg_match('/^I18N::number\((.+?)(,([\d+]))?\)$/', $display, $match)) {
+                    			$display = I18N::number($match[1], isset($match[3]) ? $match[3] : 0);
```

Comment:
Can you convert spaces to tabs for indentation.
http://wiki.webtrees.net/en/Coding_Standards

Maybe make the second pattern non-capturing.  e.g. `(,` becomes `(?:,` - or maybe even `(?:, *`